code/modules/mob/living/carbon/human/species/emote-synthetic.dm

```diff
@@ -6,55 +6,349 @@
+	sound = 'sound/voice/joe/beyond_repair.ogg'
+	say_message = "Hmm, far beyond repair."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/come_out_vent
+	key = "comeoutvent"
+	sound = 'sound/voice/joe/come_out_vent.ogg'
+	say_message = "Come out of the vent system, please."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/could_require_attention
+	key = "couldrequireattention"
+	sound = 'sound/voice/joe/could_require_attention.ogg'
+	say_message = "This could require my attention."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/dangerous_items
+	key = "dangerousitems"
+	sound = 'sound/voice/joe/dangerous_items.ogg'
+	say_message = "You are carrying some very dangerous items."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/day_never_done
+	key = "dayneverdone"
+	sound = 'sound/voice/joe/day_never_done.ogg'
+	say_message = "A synthetic's day is never done."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/detailed_report
+	key = "detailedreport"
+	sound = 'sound/voice/joe/detailed_report.ogg'
+	say_message = "APOLLO will require a detailed report."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/fire
+	key = "fire"
+	sound = 'sound/voice/joe/fire.ogg'
+	say_message = "Only wild animals fear fire."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/fire_drill
+	key = "firedrill"
+	sound = 'sound/voice/joe/fire_drill.ogg'
+	say_message = "Please congregate at your nearest fire assembly point. This is not a drill; do not panic."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/unprotected_flames
+	key = "unprotectedflames"
+	sound = 'sound/voice/joe/unprotected_flames.ogg'
+	say_message = "Unprotected flames are extremely dangerous and entirely unadvisable."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/follow_me
+	key = "followme"
+	sound = 'sound/voice/joe/follow_me.ogg'
+	say_message = "Follow me."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/further_assistance
+	key = "furtherassistance"
+	sound = 'sound/voice/joe/further_assistance.ogg'
+	say_message = "Please call if you need further assistance."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/good_day
+	key = "goodday"
+	sound = 'sound/voice/joe/good_day.ogg'
+	say_message = "Good day."
+	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
+
+/datum/emote/living/carbon/human/synthetic/working_joe/health_risks
+	key = "healthrisks"
+	sound = 'sound/voice/joe/health_risks.ogg'
+	say_message = "This items carry notable health risks."
```

Comment:
It should say "These items carry notable health risks." as of right now it doesnt really make sense

Reply:
ack, fixed